openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -396,11 +396,15 @@ def retirement_queue(self, request):
         """
         try:
             cool_off_days = int(request.GET['cool_off_days'])
-            states = request.GET['states'].split(',')
-
             if cool_off_days < 0:
                 raise RetirementStateError('Invalid argument for cool_off_days, must be greater than 0.')
 
+            if 'states' in request.GET.dict().keys():
+                # Use all the "states" passed-in - not just the last-passed.
+                states = [p[1] for p in request.GET.lists() if p[0] == 'states'][0]
```

Comment:
I think you can just do `request.GET.getlist('states')` here: https://docs.djangoproject.com/en/1.11/ref/request-response/#django.http.QueryDict.getlist

Reply:
Ah - didn't see that convenience method. I'll try it and commit the change if it works.